modules/install-sno_additional-requirements-for-installing-on-a-single-node-on-aws.adoc

```diff
@@ -0,0 +1,18 @@
+// This module is included in the following assemblies: 
+//
+// installing/installing_sno/install-sno-preparing-to-install-sno.adoc
+
+:_content-type: CONCEPT
+[id="additional-requirements-for-installing-on-a-single-node-on-aws_{context}"]
+= Additional requirements for installing on a single node on AWS
+
+The AWS documentation for installer-provisioned installation is written with a high availability cluster consisting of three control plane nodes. When referring to the AWS documentation, consider the differences between the requirements for a {sno} cluster and a high availability cluster.
+
+* The required machines for cluster installation in AWS documentation indicates a temporary bootstrap machine, three control plane machines, and at least two compute machines. You require only one AWS instance for the control plane node and no worker nodes.
+
+* The minimum resource requirements for cluster installation in the AWS documentation indicates a control plane node with 4 vCPUs and 100GB of storage. For a single node cluster, you must have a minimum of 8 vCPU cores and 120GB of storage.
```

Comment:
We have maintained a minimum resource table in [1], since they are the same, I suggest keeping [1] only, wdyt?

[1] https://57452--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_sno/install-sno-preparing-to-install-sno.html#install-sno-requirements-for-installing-on-a-single-node_install-sno-preparing